apps/designer/app/designer/features/style-panel/shared/property-name.tsx

```diff
@@ -14,8 +15,126 @@ import {
   Separator,
 } from "@webstudio-is/design-system";
 import { UndoIcon } from "@webstudio-is/icons";
+import type { Instance } from "@webstudio-is/react-sdk";
+import { utils } from "@webstudio-is/project";
 import { isFeatureEnabled } from "~/shared/feature-flags";
-import { getStyleSource, StyleInfo } from "./style-info";
+import { useSelectedInstanceData } from "~/designer/shared/nano-states";
+import { useBreakpoints, useRootInstance } from "~/shared/nano-states";
+import { type StyleInfo, type StyleSource, getStyleSource } from "./style-info";
+
+const PropertyPopoverContent = ({
+  properties,
+  style,
+  styleSource,
+  onReset,
+}: {
+  properties: StyleProperty[];
+  style: StyleInfo;
+  styleSource: StyleSource;
+  onReset: () => void;
+}) => {
+  const [breakpoints] = useBreakpoints();
+  const [rootInstance] = useRootInstance();
+  const [selectedInstanceData] = useSelectedInstanceData();
+  const selectedInstanceId = selectedInstanceData?.id;
+
+  // @todo replace with normalized data access
+
+  const breakpointsMap = useMemo(() => {
```

Comment:
same here, I would think this is not needed optimization, both building a map and using useMemo, because its like 5 items most of the time

Reply:
Will be removed eventually

Reply:
I know, but we don't need that code, its optimizing nothing and its like 10-15LOC that does nothing but noise